### PR TITLE
Clojure 1.9 and 1.10 profiles

### PR DESCRIPTION
**Pull request by jeremyvdw**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -7,7 +7,9 @@
                  [com.stuartsierra/component "0.3.1"]
                  [ring/ring-jetty-adapter "1.5.0"]
                  [suspendable "0.1.1"]]
-  :aliases {"test-all" ["with-profile" "default:+1.8" "test"]}
+  :aliases {"test-all" ["with-profile" "default:+1.8" "default:+1.9" "default:+1.10" "test"]}
```
Comment (weavejester) [inline project.clj]: This should be: `"default:+1.8:+1.9:+1.10"`. The colon separates profiles and executes them sequentially.

**Review by weavejester — CHANGES_REQUESTED** [review]
Thanks for the PR, but the "test-all" alias is set up incorrectly.

File: project.clj (hunk 1 above)
Comment (jeremyvdw) [reply project.clj]: Thanks I didn't know that

**Comment by jeremyvdw** [comment]
@weavejester I force-pushed the change to keep the git history clean

**Comment by weavejester** [comment]
Thanks! Can you change the commit message to:

    Add test profiles for Clojure 1.9 and 1.10

**Comment by jeremyvdw** [comment]
Done, thanks James.

**Pull request merged**